### PR TITLE
docs: fix doc build

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

It's affected by
https://github.com/medfreeman/markdown-it-toc-and-anchor/commit/3eb39de83686fbb79f6a2197ef7737104470f289

### CODE REVIEW

File: docs/source/zh-cn/core/cookie-and-session.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 title: cookie 与 session
 ---
 
-## cookie
+### cookie
```
Comment (popomore) [inline docs/source/zh-cn/core/cookie-and-session.md]: 没看懂具体逻辑，不过这样改就好了

Comment (atian25) [inline docs/source/zh-cn/core/cookie-and-session.md]: 呃，其他文件呢

**Comment by popomore** [comment]
就这个有问题
TZ | 天猪 <notifications@github.com>于2017年3月6日 周一21:29写道：

**Comment by fengmk2** [comment]
这么神奇

**Pull request merged**